install.sh

```diff
@@ -19,13 +19,13 @@ installandconfigure(){
     sudo make install
 
     mkdir -p ~/.config/custard
-    cp ./examples/sxhkdrc ~/.config/custard/sxhkdrc
+    cp ./examples/sxhkdrc ~/.config/sxhkd/sxhkdrc
```

Comment:
sxhkdrc should be in the custard configuration directory with the other configurations for custard. Users with sxhkd most likely had it installed prior to custard, even more likely is that they had it installed for bspwm, in which case this line would override their bspwm keybindings.